### ISSUE TITLE
CosmJS cannot be used with CRA/react-scripts version 5 (Webpack 5)

### ISSUE THREAD

**Issue by webmaster128** [cosmos/cosmjs#1389]
When upgrading react-scripts from v4 to v5, you also get a webpack v4 to v5 upgrade. In Webpack 5 a bunch of modules need to be configures manually. CosmJS lists the required condiguration here: https://github.com/cosmos/cosmjs#webpack-configs.

However, since Create React App/react-scripts don't allow you to easily create your own webpack 5 config, this does not work together.

**Comment by mrakus-sofomo** [comment]
Did you find any option to not use crypto-browserify, stream-browserify and make cosmjs run?

**Comment by webmaster128** [comment]
Given the responses in https://twitter.com/simon_warta/status/1634969846676078592 I assume CRA is dead and folks use different frameworks anyways.

**Comment by mrakus-sofomo** [comment]
However, it seems to me that this does not solve the problem. Let's imagine a situation in which we have a package that is used by our customers. If I want to use @cosmjs, then I have to require each client to install polyfills.

**Comment by webmaster128** [comment]
Nope, you just have to tell Webpack to not include those packages. This is how things are supposed to work. `require("not-found")` just throws at runtime and CosmJS is fine with that as there are different implementations.
